lib/ex_webrtc/rtp/jitter_buffer.ex

```diff
@@ -0,0 +1,166 @@
+defmodule ExWebRTC.RTP.JitterBuffer do
```

Comment:
In the case of saving to a file, it might not make a difference. Still, when a player is on the receiving end, assuming we returned the very first packets right away, the player would need to handle the possibility of a `latency` long gap between two packets (so `latency` long freeze). When the initial latency is forced, we might store a bunch of packets that came earlier than the initial latency and then dump them all at once after the initial latency, but as long as there are no gaps longer than `latency`, no freezes will occur and RTP timestamps pace the playout anyway.

Does it make sense?

Reply:
Hmmmmm. What about this: if we don't postpone the playback at the beginning, we will do this on the first detected gap. So we either wait at the beginning or during the first detected gap. Is that the whole difference?

Reply:
Ye, but you'd rather have an initial latency than freezes through the playback. Lack of the initial latency kinda defeats the whole purpose of a jitter buffer.

Reply:
Agree 👍